tests/php/test-amp-audio-converter.php

```diff
@@ -235,6 +235,7 @@ public function test_converter( $source, $expected = null, $args = [] ) {
 		}
 		$dom = AMP_DOM_Utils::get_dom_from_content( $source );
 		if ( ! empty( $args['add_dev_mode'] ) ) {
+			$dom->ampDevModeActive = true; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
```

Comment:
I don't think this is right. This property should only be relevant when `loadHTML` first parses the document. The `ampDevModeActive`  property on the document should be read-only. It's kinda indicating the flavor of the HTML document.

Reply:
Applied suggestion in https://github.com/ampproject/amp-wp/pull/4440/commits/fa005500c92f503c9dc0b81ebccd598f8f2d1ba5.